inst/acronyms_styles.lua

```diff
@@ -74,6 +74,18 @@ styles["short-long"] = function(acronym, insert_links)
     return create_element(text, acronym.key, insert_links)
 end
 
+-- First use: long name
+-- Next use: long name
+styles["long-long"] = function(acronym, insert_links)
+    local text
+    if acronym:isFirstUse() then
+        text = acronym.longname
+    else
+        text = acronym.longname
+    end
```

Comment:
As both first and non-first ("next") uses result in using the longname, this if can be simplified.

Reply:
```suggestion
    text = acronym.longname
```

Reply:
Perhaps a comment can be added to explain, but I am not sure whether this is necessary, as the function clearly indicates that it returns the longname in both cases.